common/model-views.cpp

```diff
@@ -1911,30 +1926,51 @@ namespace rs2
         return results;
     }
 
-    bool is_resolution_max(const std::vector<std::pair<int, int>>& res_values, int width, int height, const std::pair<int, int>& calibration_res)
+    bool subdevice_model::is_d405_oem_cal_profile(int d405_calibration_width, int d405_calibration_height) const
+    {
+        // checking format
+        bool is_cal_format = false;
+        for (auto it = stream_enabled.begin(); it != stream_enabled.end(); ++it)
+        {
+            if (it->second)
+            {
+                auto format = format_values.at(it->first)[ui.selected_format_id.at(it->first)];
+                if (format == RS2_FORMAT_Y16)
+                {
+                    is_cal_format = true;
+                    break;
+                }
+            }
+        }
+        if (is_cal_format)
+        {
+            // checking resolution
+            auto width = res_values[ui.selected_res_id].first;
+            auto height = res_values[ui.selected_res_id].second;
+            bool is_cal_res = (width == d405_calibration_width && height == d405_calibration_height);
```

Comment:
Redundant, the calibration resolution is unique - the `bool` can be dropped

Reply:
Done